### ISSUE TITLE
[a11y] Material Chip delete icon should be focusable by voiceover

### ISSUE THREAD

**Issue by clocksmith** [flutter/flutter#48739]
Currently, only the whole chip gets focused by voiceover. When there is a delete icon, that should be separate from the primary action (or label) for the whole chip.
